ddprof-lib/src/main/cpp/objectSampler.cpp

```diff
@@ -188,14 +189,16 @@ Error ObjectSampler::updateConfiguration(u64 events, double time_coefficient) {
       CONFIG_UPDATE_CHECK_PERIOD_SECS, 15);
 
   float signal = pid_controller.compute(events, time_coefficient);
```

Comment:
minor, there is already a conversion here from double to float. Then a truncation to an int.